dashboard/app/assets/javascripts/soundEffects.js

```diff
@@ -33,6 +61,26 @@ function Sounds() {
   this.soundsById = {};
 }
 
+/**
+ * @param {Array.<string>} filenames ending in (.mp3|.ogg|.wav)
```

Comment:
Clarified a bit, added note about repeat extensions, the current behavior of overwriting feels OK since the intent of providing different paths is to have as many file format options as possible